### PR TITLE
Bugfix/um 741 incorrect nvcc warnings about missing return statement should be supressed

### PR DESCRIPTION
**Pull request by mcfadden8**

### CODE REVIEW

**Comment by mcfadden8** [comment]
LGTM

File: examples/CMakeLists.txt [hunk 1]
```diff
@@ -111,6 +116,11 @@ if (ENABLE_CUDA)
     multi_device
     PROPERTIES CUDA_SEPARABLE_COMPILATION On)
 
+  target_compile_options(
+    multi_device PUBLIC
```
Comment (davidbeckingsale) [inline examples/CMakeLists.txt:120]: Did this not get inherited from the `umpire` target?

Comment (mcfadden8) [reply examples/CMakeLists.txt]: Over did it.  Oops.  This has been removed (nice catch).

**Comment by mcfadden8** [comment]
LGTM

**Comment by mcfadden8** [comment]
@davidbeckingsale, this is ready.

**Review by davidbeckingsale — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,9 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ### Fixed
 
+- Fixed warnings from nvcc about potential early returns from non-void
```
Comment (davidbeckingsale) [inline CHANGELOG.md:22]: You might want to say it is an erroneous warning?

**Pull request merged**